modules/scheduler/gke-cluster/README.md

```diff
@@ -42,6 +42,31 @@ are created in the VPC module. By default the `gke-cluster` module will look for
 ranges with the names `pods` and `services`. These names can be configured using
 the `pods_ip_range_name` and `services_ip_range_name` settings.
 
+### Multi-networking
+
+To create network objects in GKE cluster, you can pass a multivpc module to gke-cluster module instead of [applying a manifest manually](https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#create-gke-environment). Passing a multivpc module enables multi networking and [Dataplane V2](https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2?hl=en) on the cluster.
+
+```yaml
+  - id: network
+    source: modules/network/vpc
```

Comment:
I think you need more fields here to make it compatible with GKE. See your example.